### DISCUSSION TITLE
Multiprice events reject waiting list

### DISCUSSION THREAD

**Discussion by StickyPixels** [liedekef/events-made-easy#406] [Q&A] [answered]
Hi Franky,

I have set up events with different price categories with a specific number of seats per category. 
Now when I try to create a waitinglist for the event, this value gets reset to zero upon Save. No matter if I specify a single value or values per price category, it is not possible to create a waiting list for multiprice events.

It's not a big issue, and there are workarounds, but I don't get the rationale behind this behavior. Can you illuminate?

Thanks again, cheers,
Leo

**Comment by liedekef** [comment, marked as answer]
The waiting list feature currently doesn't support different seat categories, so if the event is multiseat it will always be reset to 0
